tests/limits/test_vif_limit.py

```diff
@@ -0,0 +1,85 @@
+import ipaddress
+import os
+import pytest
+import logging
+
+# Requirements:
+# - one XCP-ng host (--host) >= 8.2
+# - a VM (--vm)
+
+vif_limit = 16
+interface_name = "enX"
+vcpus = '8'
+
+# There is a ResourceWarning due to background=True on an ssh call
+# We do ensure the processes are killed
+@pytest.mark.filterwarnings("ignore::ResourceWarning")
+@pytest.mark.debian_uefi_vm
```

Comment:
Why specifically UEFI?

Reply:
the only thing this really needs to express is "non-alpine", preferably debian because some other distros might have different naming schemes for interfaces. uefi is not necessary